src/lib/include/roc/address.h

```diff
@@ -85,6 +85,22 @@ typedef struct roc_address {
 ROC_API int
 roc_address_init(roc_address* address, roc_family family, const char* ip, int port);
 
+/** Set multicast interface address.
+ *
+ * @b Parameters
+ *  - @p address should point to a properly initialized address struct
+ *  - @p family should be @c ROC_AF_AUTO, @c ROC_AF_IPv4, or @c ROC_AF_IPv6
+ *  - @p iface should point to a zero-terminated string with a valid IPv4 or
+ *    IPv6 multicast address
```

Comment:
`IPv4 or IPv6 multicast address` -> `IPv4 or IPv6 address`

Also suggest to rename `iface` to `iface_ip`.

And also please add some remarks, something like:

```
@remarks
  This function assigns a multicast interface to an address. If the address
  has a multicast interface, roc_receiver_bind() will use it to join the
  multicast group on that interface. Otherwise, roc_receiver_bind() will not
  join the multicast group and the user should do it manually in order to be
  able to receive multicast packets.

  @p iface_ip should be an IP address of the network interface on which to
  join to the multicast group. It may be "0.0.0.0" or "[::]" to join
  the multicast group on all available interfaces.
```

Reply:
We should also explain this in the manual page for roc-recv.

Reply:
Outdated

Reply:
> Outdated

Not quite. I think we still need to add an explanation to the manual page.

Reply:
I hope I understood you correctly, see https://github.com/roc-project/roc/blob/81baa7f6f90080b39e7fba604eb134d38575ce76/docs/sphinx/manuals/roc_recv.rst

Reply:
It is also need to be added to the help page of `roc-recv`.